src/sentry/api/serializers/models/group.py

```diff
@@ -219,6 +246,9 @@ def get_attrs(self, item_list, user):
                 'resolution': resolution,
                 'resolution_actor': resolution_actor,
                 'share_id': share_ids.get(item.id),
+                'times_seen': times_seen.get(item.id, 0),
+                'first_seen': first_seen.get(item.id),  # TODO: missing?
+                'last_seen': last_seen.get(item.id),
```

Comment:
There's a potential race condition here (this is throttled by buffer flushing, IIRC?) that I don't have any good ideas how to deal with. Returning `None` seems … wrong?

Reply:
Yeah, the only other real alternative then is doing `timezone.now()` which is also wrong.

I think the consumer just needs to be able to handle `None`.